src/blog/2023/010-reworked-element-selection.md

```diff
@@ -0,0 +1,47 @@
+---
+
+title: Reworked Element Selection
+description: "Learn how the latest releases of our BPMN and DMN editors substantially rework diagram selection."
+layout: blogpost.hbs
+slug: 2023-reworked-element-selection
+preview_image: http://bpmn.io/assets/attachments/blog/2023/010-selection-rework.png
+url: https://bpmn.io/blog/posts/2023-reworked-element-selection.html
+author:
+- Nico Rehwaldt <https://github.com/nikku>
+published: 2023-11-06 12:00
+
+releases:
+  - 'bpmn-js@15.0.0'
+  - 'dmn-js@14.7.0'
+  - 'diagram-js@12.6.0'
+
+---
+
+<p class="introduction">
+  The latest releases ship of our [BPMN](https://github.com/bpmn-io/bpmn-js) and [DMN](https://github.com/bpmn-io/bpmn-js) toolkits ship with a reworked selection UI. That improved UI makes diagram element selection more ergonomic and less verbose.
+</p>
+
+<!-- continue -->
+
+Selection is an integral part of the core modeling UX. It has to work; you come across it immediately, and it leaves a lasting impression. Recent releases of our modeling toolkit simplify selection following the selected element's shape. We think the change results in a less verbose look and feel and clearly distinguishes single-element selections and multi-selection:
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/s/start">
+    <img src="{{ assets }}/attachments/blog/2023/010-selection-rework.gif" alt="Improved selection in BPMN">
+  </a>
+
+  <p class="caption">
+    Enjoy reworked selection in BPMN and DMN editing.
+  </p>
+</div>
+
+Try it on [demo.bpmn.io](https://demo.bpmn.io/s/start).
+
+
+## Wrapping up
+
+As users of our [demo](https://demo.bpmn.io) may have noticed, the reworked popup menu is only the first step on our journey to incorporate [create/append anything](https://bpmn-io.github.io/bpmn-js-connectors-extension/?aa=1) into our core. Stay tuned for future exciting updates in core modeling.
```

Comment:
Looks like a WIP copy-paste.

Reply:
I'll fix it :).

Reply:
Fixed via https://github.com/bpmn-io/bpmn.io/pull/182/commits/56bf29cddb8a9744e274238b3cf32fbb5e984280.